Assets/Code/src/Runtime/Match/Components/MatchCountdownController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace HouraiTeahouse.FantasyCrescendo.Matches
+{
+
+public class MatchCountdownController : MonoBehaviour
+{
+    public Text TextUI;
```

Comment:
We're trying to avoid use of normal Text objects when possible when we have TextMeshPro available. Please replace with a TextMeshProUGUI reference.